src/context-providers/builder/provider.tsx

```diff
@@ -1,18 +1,19 @@
+import { noop } from "lodash";
```

Comment:
use sub-directory import to reduce bundle size

```suggestion
import noop from "lodash/noop";
```